src/pages/Player.vue

```diff
@@ -224,6 +250,12 @@ export default {
       // styling class for the minimize button
       maximizeButtonClass:
         "bg-primary hover:bg-primary-hover p-1 pl-4 pr-4 sm:p-2 sm:pl-6 sm:pr-6 lg:p-4 lg:pl-6 lg:pr-6 rounded-md shadow-xl disabled:opacity-50 disabled:pointer-events-none",
+      numOfCorrect: 0, // number of correctly answered questions (valid for MCQ only)
+      numOfWrong: 0, // number of wrongly answered questions (valid for MCQ only)
+      numOfSkipped: 0, // number of skipped questions
```

Comment:
we have been following `numX` format - let's stick to that instead of `numOfX` or `numberOfX`
<img width="266" alt="Screenshot 2021-09-16 at 4 44 04 PM" src="https://user-images.githubusercontent.com/13185699/133607449-2ee5e94b-f450-4e39-97f6-32a53d3fcddd.png">